database/migrations/default/1737481505501_specify-schema-for-jurisdictions/up.sql

```diff
@@ -0,0 +1,78 @@
+drop trigger if exists afd_incidents_trigger_insert on public.afd__incidents;
+
+drop trigger if exists afd_incidents_trigger_update on public.afd__incidents;
+
+
+drop function public.afd_incidents_trigger ();
+
+create or replace function public.afd_incidents_trigger()
+returns trigger
+language plpgsql
+as $function$
+BEGIN
+  WITH jurisdiction_union AS (
+    SELECT ST_Union(geometry) AS geometry
+    FROM geo.jurisdictions
+    WHERE jurisdiction_label = 'AUSTIN FULL PURPOSE'
+  )
```

Comment:
Unioning the two records we keep from CTM that reflect this single piece of geometry

Reply:
Cool. The [`ems_incidents_trigger`](https://github.com/cityofaustin/vision-zero/blob/f9ac8dd7b609572b815b14d1e841777eb2831907/database/migrations/default/1736197283308_jurisdiction_refactor/up.sql#L157-L161) function does not do this union. does that look like a bug to you? 🤔

Reply:
I do think this is a bug, and it is valid SQL so it's not croaking. It is picking one portion of the geometry or other at random and saving the `austin_full_purpose` flag based on that portions of the geometry's containment of the incident or not. Do you want me to fold a fix into this PR @johnclary?

Reply:
yes, please, would you fix that function, Frank? I'm really glad you caught this!

good call on the backfill. we should be able to rerun [these update statements](https://github.com/cityofaustin/atd-data-tech/issues/20461#issuecomment-2576082208) for that 👍